File: Arcade/Intro/Edge of the Ocean/CplusPlus/adjacentElementsProduct.cpp
int adjacentElementsProduct(std::vector<int> inputArray)
{
    int a = inputArray.size() - 1;
    int p = -1000;
    int x = 0;
    int y = 0;

    for (int i = 0; i < a; i++)
    {

        x = inputArray[i];
        y = inputArray[i + 1];

        if (x * y > p)
        {
            p = x * y;
        }
    }
    return p;
}

File: Arcade/Intro/Edge of the Ocean/Javascript/adjacentElementsProduct.js
function adjacentElementsProduct(inputArray) {
  var a = inputArray.length - 1;
  var p = -1000;
  var x = 0;
  var y = 0;

  for (var i = 0; i < a; i++) {
    x = inputArray[i];
    y = inputArray[i + 1];

    if (x * y > p) {
      p = x * y;
    }
  }
  return p;
}
